src/frontend/admin/admin-options.js

```diff
@@ -42,3 +42,10 @@ const init = () => {
 $(document).ready(() => {
   init();
 });
+// EJ add new tabs code
+$(document).ready(() => {
+  $('li').click(function addActive() {
+      $('li.nav-tab.nav-tab-active').removeClass('nav-tab-active');
+      $(this).addClass('nav-tab-active');
+  });
```

Comment:
You don't need to duplicate `$(document).ready` and can instead add this code block into the existing one below the `init();` line

Reply:
Even better, this code can be consolidated with the `handleTabs` logic. I can take a look at this on the weekend